srcDoc/release_notes.rst

```diff
@@ -798,6 +807,33 @@ we could get into the following error:
 Known Issues
 ------------
 
+Running into SHMEM error on CPP Modern
```

Comment:
Shared Memory issues when running the Modern C++ API Implementation.

Reply:
Done
Without the "." because it is a title.